arcade/cinematic/default.py

```diff
@@ -0,0 +1,52 @@
+from typing import Optional, Tuple, Iterator, TYPE_CHECKING
+from contextlib import contextmanager
+
+from pyglet.math import Mat4
+
+from arcade.cinematic.types import Projector
+from arcade.window_commands import get_window
+if TYPE_CHECKING:
+    from arcade.application import Window
+
+
+class DefaultProjector:
```

Comment:
Make it _DefaultProjecter? if you do not want it to be used by a developer.

Reply:
Good idea, thank you!